docs/server/source/server-reference/configuration.md

```diff
@@ -248,6 +251,38 @@ export BIGCHAINDB_WSSERVER_PORT=9985
 }
 ```
 
+## wsserver.advertised_scheme, wsserver.advertised_host and wsserver.advertised_port
+
+These settings are for the advertising the Websocket URL to external clients in
+the root API endpoint. These configurations might be useful if your deployment
+is hosted behind a firewall, NAT, etc. where the exposed public IP or domain is
+different from where BigchainDB is running.
+
+**Example using environment variables**
+```text
+export BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME=wss
+export BIGCHAINDB_WSSERVER_ADVERTISED_HOST=mybigchaindb.com
+export BIGCHAINDB_WSSERVER_ADVERTISED_PORT=443
+```
+
+**Example config file snippet**
+```js
+"wsserver": {
+    "advertised_scheme": "wss",
+    "advertised_host": "mybigchaindb.com",
+    "advertised_port": 443
```

Comment:
In BigchainDB Server version 1.0.0, we already had `wsserver.scheme`, `wsserver.host` and `wsserver.port` as configuration settings. This PR adds three more: `wsserver.advertised_scheme`, `wsserver.advertised_host` and `wsserver.advertised_port`. I like the fact that they're at the same level in the tree/hierarchy as the existing settings. Flat is better than nested, etc.

It also makes sense to keep the words "advertised" and "scheme" together in a compound word-phrase. It's like New Orleans, which isn't New.Orleans (i.e. New doesn't have an Orleans). Similarly, it's not like wsserver has an advertised which has a scheme. The wsserver has an advertised_scheme.